safe/definitions/hazard_classifications.py

```diff
@@ -4,7 +4,7 @@
 See https://github.com/inasafe/inasafe/issues/2920#issuecomment-229874044
 to have a table showing you classes of each kind of hazard.
 """
-from safe.definitions import concepts, small_number
```

Comment:
Yes, I agree. I will put in the `concepts.py` like @timlinux said in the stand up.